### PR TITLE
Add NewsletterStatus component [MAILPOET-2999]

### PR DESCRIPTION
**Pull request by amine-mp**
[MAILPOET-2999]

[MAILPOET-2999]: https://mailpoet.atlassian.net/browse/MAILPOET-2999

### CODE REVIEW

File: views/layout.html [hunk 1]
```diff
@@ -94,6 +94,8 @@
   'setFromAddressEmailSuccess': __('Excellent. Your authorized email was saved. You can change it in the [link]Basics tab of the MailPoet settings[/link].', 'mailpoet'),
   'setFromAddressEmailNotAuthorized': __('Can’t use this email yet! [link]Please authorize it first[/link].', 'mailpoet'),
   'setFromAddressEmailUnknownError': __('An error occured when saving FROM email address.', 'mailpoet'),
+
+  'notSentYet': __('Not senrt yet!'),
```
Comment (wxa) [inline views/layout.html:98]: Typo in the string: `senrt` => `sent`.

File: assets/js/src/common/listings/_stories/newsletter_status.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { addHours, subHours } from 'date-fns';
+import MailPoet from 'mailpoet';
+import NewsletterStatus from '../newsletter_status';
+
+MailPoet.I18n.add('notSentYet', 'Not sent yet!');
+
+export default {
+  title: 'NewsletterStatus',
```
Comment (wxa) [inline assets/js/src/common/listings/_stories/newsletter_status.tsx:9]: The Storybook group should be `Listing`.

File: assets/js/src/common/listings/newsletter_status.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  const filled = perimeter * (percentage / 100);
+  const empty = perimeter - filled;
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+    >
+      <circle fill="none" cx="12" cy="12" r="8" stroke="#E5E9F8" />
```
Comment (wxa) [inline assets/js/src/common/listings/newsletter_status.tsx:66]: Should the stroke color be set in the CSS by a variable?

Comment (amine-mp) [reply assets/js/src/common/listings/newsletter_status.tsx:66]: @NeosinneR This is the color of the background circle. Do you think we need to make it customizable?

Comment (NeosinneR) [reply assets/js/src/common/listings/newsletter_status.tsx:66]: Let's style it in CSS (`$color-tertiary-light`), small color changes in the future will be easier.

**Pull request merged**